### PR TITLE
Make compatible with various versions of unity

### PR DESCRIPTION
**Pull request by mreram**

### CODE REVIEW

File: auth-legacy/build.gradle [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+ext {
+    package_id = 'BazaarAuth'
+    artifact_id = 'auth-legacy'
+}
+
+android {
+    compileSdkVersion 29
```
Comment (shayanpourvatan) [inline auth-legacy/build.gradle:11]: Move this variable in another file to reuse it in another Gradle file

File: auth-legacy/build.gradle [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    buildToolsVersion "29.0.3"
+
+    defaultConfig {
+        minSdkVersion 16
+        targetSdkVersion 29
```
Comment (shayanpourvatan) [inline auth-legacy/build.gradle:16]: Same for these too

Comment (mreram) [reply auth-legacy/build.gradle]: Done

File: auth-legacy/build.gradle (hunk 1 above)
Comment (mreram) [reply auth-legacy/build.gradle]: Done

**Pull request merged**